experimental/SetPartitions/src/AbstractPartition.jl

```diff
@@ -0,0 +1,18 @@
+"""
+AbstractPartition
+
+Abstract type of Partition for classical-, colored- and spatial Partitions.
```

Comment:
Hmm, "Abstract type of Partition" doesn't parse as a valid english sentence for me.  Also, if intended to be used as a noun, it should be in lowercase; if it is meant to refer to a type it should be surrounded by backticks.